conf/app.ini

```diff
@@ -71,6 +84,7 @@ MACARON             = file
 
 [log.file]
 FILE_NAME = gitea.log
+LEVEL = Warn
```

Comment:
Did you test fail2ban before to commit this ? Because sometime we use a specific log level to make fail2ban work correctly

Reply:
Indeed, that was a problem !